src/metabase/api/search.clj

```diff
@@ -163,7 +163,7 @@
 ;; Databases can't be archived
 (defmethod archived-where-clause (class Database)
   [model archived?]
-  [:= 1 1])
+  [:= 1 (if archived? 2 1)])
```

Comment:
@tsmacdonald wanted your eyes specifically on this part. There were databases on the archive page which was quite strange since they cannot be archived. Seems like we would never want databases in archived results

Reply:
Yeah, this seems sensible (thanks for the work!). Some of my early search work was completely unaware of the archived page being powered by the search endpoint (I still think is not the best idea), so when I added databases to the search results I wasn't thinking about it.

Reply:
I agree. I was quite surprised to see that we were scoring and ranking these things.